lib/radiator/directory/editor/manager.ex

```diff
@@ -67,6 +88,10 @@ defmodule Radiator.Directory.Editor.Manager do
     |> Repo.update()
   end
 
+  def delete_audio(%Audio{} = audio) do
+    Repo.delete(audio)
```

Comment:
does this cascade correctly? e.g. is the audio removed from all entities it is associated with and does it delete an audio_file if it is the only user?

Reply:
associated `audio_file`s are deleted via Postgres constraints set in the migration. I wasn't sure so I added a test to verify.

It also deletes the associated episode, which feels to destructive. I'm going to change it from `:delete_all` to `:nilify_all` there.